ironfish-cli/src/commands/service/snapshot.ts

```diff
@@ -71,20 +76,13 @@ export default class CreateSnapshot extends IronfishCommand {
       required: false,
       description: 'The path where the snapshot should be saved',
     }),
-    maxBlocksPerChunk: Flags.integer({
-      char: 'm',
-      required: false,
-      default: isNaN(Number(process.env.MAX_BLOCKS_PER_SNAPSHOT_CHUNK))
-        ? 1000
-        : Number(process.env.MAX_BLOCKS_PER_SNAPSHOT_CHUNK),
-      description: 'The max number of blocks per file in the zipped snapshot',
-    }),
   }
 
   async start(): Promise<void> {
     const { flags } = await this.parse(CreateSnapshot)
 
     const bucketUrl = (flags.bucketUrl || DEFAULT_SNAPSHOT_BUCKET_URL || '').trim()
+    const snapshotFileName = (flags.snapshotFileName || DEFAULT_SNAPSHOT_FILE_NAME).trim()
```

Comment:
I think we can get rid of this config, and just use a name here so on disk it has a static name, but in the bucket the snapshot should probably use the timestamp like it was originally.